README.rst

```diff
@@ -107,11 +256,6 @@ Configuration
 Operations
 ==========
 
-Bootstrapping Consumers
------------------------
-
-.. todo:: Rewrite this as part of the tutorial using pgbench after the replication worker is done.
-
```

Comment:
I am not totally sure I agree with moving images around, but YMMV.

I think this is a good, clear example: https://github.com/tkaemming/bottledwater-pg/blob/compose/README.md#running-in-docker